model/jpa/src/main/java/org/keycloak/models/jpa/entities/RealmEntity.java

```diff
@@ -137,7 +137,7 @@
     @Column(name="EMAIL_THEME")
     protected String emailTheme;
 
-    @OneToMany(cascade ={CascadeType.REMOVE}, orphanRemoval = true, mappedBy = "realm")
+    @OneToMany(cascade ={CascadeType.REMOVE}, orphanRemoval = true, mappedBy = "realm", fetch = FetchType.EAGER)
```

Comment:
Yes, as otherwise I get a lazy load error in some tests. Would we ever load realms without attributes? I don't get why this is being lazy loaded in the first place.

Reply:
I'm just afraid that this can impact somehow performance when loading realms from the database. If we are sure that attributes are needed when loading a realm, that is OK.

But if this is not always true, I'm not sure about the impacts. We already have issues when having a lot of realms, so that could make things even worse?

If this was set to EAGER already and you were changing to LAZY, I would have the same considerations.

So, if you really think this is not a problem. I'm OK with it.